modules/core/src/main/scala/com/azavea/stac4s/extensions/eo/EOItemExtension.scala

```diff
@@ -8,7 +8,6 @@ import io.circe.syntax._
 import com.azavea.stac4s.extensions.ItemExtension
 
 case class EOItemExtension(
-    gsd: Double,
```

Comment:
Change #3: `gsd` is no longer a part of the extension, it is a part of `item.properties`